packages/backend/apps/multitenancy/models.py

```diff
@@ -0,0 +1,42 @@
+import hashid_field
+
+from django.db import models
+from django.conf import settings
+from django.utils.text import slugify
+
+from . import constants
+from .managers import TenantManager
+
+
+class Tenant(models.Model):
+    id: str = hashid_field.HashidAutoField(primary_key=True)
+    creator: settings.AUTH_USER_MODEL = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    name: str = models.CharField(max_length=100, unique=True)
+    slug: str = models.SlugField(max_length=100, unique=True)
+    type: str = models.CharField(choices=constants.TenantType.choices)
+    created = models.DateTimeField(auto_now_add=True)
```

Comment:
I think it should be `created_at`?

and `updated_at` while you're at it. I wonder if we don't have a mixin for this already. If not, pretty sure it's a good idea to introduce it.